wordstats/tests/test_cognate.py

```diff
@@ -0,0 +1,85 @@
+from unittest import TestCase
+
+from wordstats.loading_from_hermit import load_language_from_hermit
+from wordstats.edit_distance_absolute import WordDistanceAbsolute
+from wordstats.edit_distance import WordDistanceEdit
+from wordstats.edit_distance_overlap import WordDistanceOverlap
+
+from wordstats.cognate_files_path import *
+
+from wordstats.cognate_info import CognateInfo
+class CognateTests(TestCase):
+
+    @classmethod
+    def testload(self):
+
+        info = CognateInfo.load_from_path("de", "nl","edit_distance")
+
+
+    @classmethod
+    def testeditDistanceAbsolute(self):
+        german = list(load_language_from_hermit("de").word_info_dict.keys())
+        dutch = list(load_language_from_hermit("nl").word_info_dict.keys())
+
+        cognateFM = CognateInfo("de", "nl","absolute")
+
+        distanceMetric = WordDistanceAbsolute()
+        distanceMetric.loadConfig("de", "nl", "absolute")
+        #if load fails create empty config file and warn user
+
+
+        cognateFM.apply_distance_metric(german, dutch, distanceMetric.edit_distance)
+        cognateFM.save_candidates()
```

Comment:
i am a bit puzzled about these tests. 
what are they doing? 
they have no assertions in them.

also, it seems that they modify the language_data folder. 
this is very risky. 

tests should never interfere with the actual data.

imagine that on the deployed server, somebody runs the
tests. that would destroy the valuable cognate information.

if i am right, we have to discuss about testing!